lib/menu.rb

```diff
@@ -0,0 +1,39 @@
+require 'order'
+require 'takeaway'
+
+class Menu
+  # Dish class here is like Menu of dishes
+  attr_reader :dishes
+
+  def initialize(dishes = [{ name: 'chinese', price: 8 }, 
+  { name: 'fishandchips', price: 7 }, 
+  { name: 'chicken', price: 5 }, { name: 'beef', price: 6 }])
+    @dishes = dishes
+  end 
+
+  def display_list
+    @dishes.each do |x|
+      puts "#{x[:name]} #{x[:price]}"
+     # how to make this a return statement and not puts
+    end 
+  end 
+
+  def dish?(dish)
```

Comment:
Maybe this method should be made private. It is only used by this class and maybe you don't need to have it publicly accessible